packages/mineral-ui-tokens/tokens/generic/textColor.json

```diff
@@ -33,10 +33,10 @@
     { "name": "color_warning_active",  "value": "{!bronze_70}" },
     { "name": "color_warning_focus",   "value": "{!bronze_60}" },
     { "name": "color_warning_hover",   "value": "{!bronze_50}" },
-    { "name": "color_grayscale",         "value": "{!gray_60}" },
-    { "name": "color_grayscalePrimary",  "value": "{!gray_90}" },
-    { "name": "color_grayscale_active",  "value": "{!gray_70}" },
-    { "name": "color_grayscale_focus",   "value": "{!gray_60}" },
-    { "name": "color_grayscale_hover",   "value": "{!gray_50}" }
+    { "name": "color_grayscale",         "value": "{!gray_80}" },
+    { "name": "color_grayscalePrimary",  "value": "{!gray_100}" },
```

Comment:
should this be grayscale_primary? looks like that's how it was but just checking